cmd/crio/main.go

```diff
@@ -71,6 +71,12 @@ func mergeConfig(config *server.Config, ctx *cli.Context) error {
 	if ctx.GlobalIsSet("listen") {
 		config.Listen = ctx.GlobalString("listen")
 	}
+	if ctx.GlobalIsSet("stream-address") {
```

Comment:
we need to add these to https://github.com/kubernetes-incubator/cri-o/blob/master/cmd/crio/config.go#L11 right?